src/Entities/Stats.java

```diff
@@ -0,0 +1,70 @@
+package Entities;
+
+import java.util.ArrayList;
+
+public class Stats {
+    private int level;
+    private String affinity;
+    private int baseHP;
+    private int baseDEF;
+    private int baseATK;
+    private int baseCRIT;
+    //constructors below
+    public Stats(int level, String affinity, int baseHP, int baseDEF, int baseATK, int baseCRIT) {
+        this.level = level;
+        this.affinity = affinity;
+        this.baseHP = baseHP;
+        this.baseDEF = baseDEF;
+        this.baseATK = baseATK;
+        this.baseCRIT = baseCRIT;
+    }
+    //getters below
+    public int getLevel() {
+        return level;
+    }
+    public String getAffinity() {
+        return affinity;
+    }
+    public int getBaseHP() {
+        return baseHP;
+    }
+    public int getBaseDEF() {
+        return baseDEF;
+    }
+    public int getBaseATK() {
+        return baseATK;
+    }
+    public int getBaseCRIT() {
+        return baseCRIT;
+    }
+    //setters below
+    public void setLevel(int level) {
+        this.level = level;
+    }
+
+    public String serializer(){
+        return level + ":" +
```

Comment:
i think same issue here. the parts that are being concatenated aren't converted to strings. this may cause compiler issue